tools/script_template.py

```diff
@@ -1,5 +1,43 @@
 import argparse
 import yaml
+import alara_output_processing as aop
+import openmc
+import numpy as np
+
+def write_out_tables(run_out, run_lbl, time_unit):
+    '''
+    Takes the output of a single ALARA run and converts output tables to Pandas DataFrame
+    inputs: 
+        run_out (path to ALARA output file)
+        run_lbl (name of the run)
+        time_unit (units for cooling time)
+    output: output_tables (Pandas DataFrame containing all output data)
+    '''
+    alara_data = aop.FileParser(run_out, run_lbl, time_unit)
+    output_tables = alara_data.extract_tables()
+    return output_tables
+
+def open_flux_file(flux_file):
+    flux_lines = open(flux_file, 'r').readlines()
```

Comment:
This is a common pattern, in part because it takes care of closing the file for you
```suggestion
    with open(flux_file, 'r') as flux_data:
        flux_lines = flux_data.readlines()
```